test/e2e/specs/editor/blocks/gallery.spec.js

```diff
@@ -136,12 +136,12 @@ test.describe( 'Gallery', () => {
 
 		await expect( gallery ).toBeVisible();
 		await editor.selectBlocks( gallery );
+		await editor.clickBlockToolbarButton( 'Add caption' );
 
 		const caption = gallery.locator(
 			'role=textbox[name="Gallery caption text"i]'
 		);
-		await expect( caption ).toBeVisible();
-		await caption.click();
+		await expect( caption ).toBeFocused();
```

Comment:
Pressing the caption button brings focus to the caption textbox automatically.
Therefore, I believe This is more reasonable than `tobeVisible`.

Reply:
Makes sense 👍